CMake/ctkMacroSetupQt.cmake.orig

```diff
@@ -0,0 +1,92 @@
+###########################################################################
```

Comment:
Do you still need this file? Appears to be an artifact leftover when you were changing things in ctkMacroSetupQt.cmake, so you should be able to remove this file.